### ISSUE TITLE
What is the correct padding side for train/eval of base model for FIM?

### ISSUE THREAD

**Issue by zhzhangcc** [deepseek-ai/DeepSeek-Coder#173]
I see in https://github.com/deepseek-ai/DeepSeek-Coder/blob/main/finetune/finetune_deepseekcoder.py#L132 that the padding side for the instruct model is set to `right`.
However, the tokenizer's default is `left. And there are no other official examples I can find for a fine-tune on the base model.

So what should the padding side by for the base model during train and eval?
